databricks_cli/secrets/cli.py

```diff
@@ -50,15 +50,31 @@
               ' principal for this option is the group "users", which contains all users in the'
               ' workspace. If not specified, the initial ACL with MANAGE permission applied to the'
               ' scope is assigned to the request issuer\'s user identity.')
+@click.option('--scope-backend-type', type=click.Choice(['AZURE_KEYVAULT', 'DATABRICKS'], case_sensitive=True),
+              default='DATABRICKS', help='The backend that will be used for this secret scope. '
+                                         'Options are (case-sensitive): 1) \'azure_keyvault\' and 2) \'databricks\' '
```

Comment:
This is very confusing, the message which shows up from help is this
```
  --scope-backend-type [AZURE_KEYVAULT|DATABRICKS]
                                  The backend that will be used for this
                                  secret scope. Options are (case-sensitive):
                                  1) 'azure_keyvault' and 2) 'databricks'
                                  (default option) Note: To create an Azure
                                  Keyvault, be sure to configure an AAD Token
                                  using'databricks-cli configure --aad-token'
```
Which one is it? Lower case or upper case?

Reply:
It is uppercase - changed the message